app/tests/beaker-plugin-safe-app/test_modules/safeApp/refreshContainersPermissions.spec.js

```diff
@@ -0,0 +1,21 @@
+let should = require('should');
+let testHelpers = require('../helpers');
+
+describe('window.safeApp.refreshContainersPermissions', () => {
+  // QUESTION: What is the purpose of refreshContainersPermissions?
```

Comment:
I guess you'd need to change them, and then pull again to get them updated? 

I'd assume the use case of an app where you request perms and then need to check if you've been given them?

@krishnaIndia can you maybe clarify here, please.

Reply:
If the permissions for an app are changed either by itself (e.g. requesting additional containers permissions), or by another app, or by another running instance of the same app, these changes are not pushed to the app, so this should allow the app to refresh them as they are kept in the safe_client_libs cache, so the cache is updated with up to date permissions.